### DISCUSSION TITLE
All about list

### DISCUSSION THREAD

**Discussion by Akyongpogi28** [community/community#49762] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I need to understand everything please everyone who can help me figure out how 🤔?

**Comment by Ntshangase** [comment]
what do you need to understand?

**Reply by eenthropy** [reply]
Plain

**Reply by Ntshangase** [reply]
What does that mean?

If you are trying to understand GitHub you can check out: https://docs.github.com/en/get-started

**Reply by anasaboayham** [reply]
Ah

**Comment by Alsaam** [comment]
Did

**Reply by Alsaam** [reply]
Did
